app/scripts/views/base.js

```diff
@@ -713,9 +713,13 @@ define(function (require, exports, module) {
      *
      * @param {String} eventName
      * @param {String} viewName
+     * @param {Object} data
      */
-    logFlowEvent (eventName, viewName) {
-      this.metrics.logFlowEvent(eventName, viewName);
+    logFlowEvent (eventName, viewName, data) {
+      this.notifier.trigger('flow.event', _.assign({}, data, {
```

Comment:
Not for this PR, but it looks like [logView, logEvent, logEventOnce, and logViewEvent](https://github.com/mozilla/fxa-content-server/blob/3495a24685f4dcd95fe3c6d82c12eb8a5bb5763d/app/scripts/views/base.js#L680:L709) could all use the same treatment.